agent/util-scripts/unittests

```diff
@@ -447,17 +454,26 @@ declare -A options=(
     [test-60]="--group=default --dir=42-iter/sample42"
     [test-61]="--group=default --dir=${_testdir}/mock-run"
     [test-62]="--group=default --dir=${_testdir}/mock-run"
+    # pbench-clear-tools
+    # Remove all tools from group good, leave default alone
+    [test-65]="--group=good"
+    # Error group does not exist
+    [test-66]="--group=bad"
+    # Remove all tools from 2 remotes, one remote does not exist
+    [test-67]="--remotes=fubar3.example.com,doesnotexist.example.com,fubar4.example.com"
+    # Remove one tool from two remotes, remotes not removed
+    [test-68]="--name=vmstat --remotes=fubar5.example.com,fubar6.example.com"
+    # One tool, one remote w label, one remote w/o label, one remote w 2 tools, 1st two remotes removed
+    [test-69]="--name=pidstat --remotes=fubar5.example.com,fubar6.example.com,fubar7.example.com"
```

Comment:
Thanks for adding the comments! That helps a lot.

I'll be really glad when we can get completely Pythonic, use pytest, and stop dealing with the gold results files, which are really awkward to review.